Makefile

```diff
@@ -1,65 +1,83 @@
 
-install: ## Install dependencies, including dev dependencies
+### Dependency commands ###
+install: ## Install script and dependencies
 	pipenv install --dev
 
-dist: ## Build docker container
-	docker build --platform linux/amd64 -t $(ECR_REGISTRY)/dspacesubmissionservice-stage:latest \
-		-t $(ECR_REGISTRY)/dspacesubmissionservice-stage:`git describe --always` \
-		-t submitter:latest .	
-
 update: install ## Update all Python dependencies
 	pipenv clean
 	pipenv update --dev
 
-publish: dist ## Build, tag and push
-	docker login -u AWS -p $$(aws ecr get-login-password --region us-east-1) $(ECR_REGISTRY)
-	docker push $(ECR_REGISTRY)/dspacesubmissionservice-stage:latest
-	docker push $(ECR_REGISTRY)/dspacesubmissionservice-stage:`git describe --always`
-
-promote: ## Promote the current staging build to production
-	docker login -u AWS -p $$(aws ecr get-login-password --region us-east-1) $(ECR_REGISTRY)
-	docker pull $(ECR_REGISTRY)/dspacesubmissionservice-stage:latest
-	docker tag $(ECR_REGISTRY)/dspacesubmissionservice-stage:latest $(ECR_REGISTRY)/dspacesubmissionservice-prod:latest
-	docker tag $(ECR_REGISTRY)/dspacesubmissionservice-stage:latest $(ECR_REGISTRY)/dspacesubmissionservice-prod:$(DATETIME)
-	docker push $(ECR_REGISTRY)/dspacesubmissionservice-prod:latest
-	docker push $(ECR_REGISTRY)/dspacesubmissionservice-prod:$(DATETIME)
 
-check-permissions-stage: ## Check infrastructure permissions on the staging deplpyment
-	aws ecs run-task --cluster dspacesubmissionservice-stage --task-definition dspacesubmissionservice-stage --network-configuration "awsvpcConfiguration={subnets=[subnet-0744a5c9beeb49a20],securityGroups=[sg-06b90b77a06e5870a],assignPublicIp=DISABLED}" --launch-type FARGATE --region us-east-1 --overrides '{"containerOverrides": [{"name": "DSS","command": ["check-permissions"]}]}'
+### Testing commands ###
+test: ## Run tests and print a coverage report
+	pipenv run coverage run --source=submitter -m pytest -vv
+	pipenv run coverage report -m
 
-check-permissions-prod: ## Check infrastructure permissions on the prod deplpyment
-	aws ecs run-task --cluster dspacesubmissionservice-prod --task-definition dspacesubmissionservice-prod --network-configuration "awsvpcConfiguration={subnets=[subnet-0744a5c9beeb49a20],securityGroups=[sg-0b29d571e70c05101],assignPublicIp=DISABLED}" --launch-type FARGATE --region us-east-1 --overrides '{"containerOverrides": [{"name": "DSS","command": ["check-permissions"]}]}'
-
-run-stage:  ## Runs the task in stage - see readme for more info
-	aws ecs run-task --cluster dspacesubmissionservice-stage --task-definition dspacesubmissionservice-stage --network-configuration "awsvpcConfiguration={subnets=[subnet-0744a5c9beeb49a20],securityGroups=[sg-06b90b77a06e5870a],assignPublicIp=DISABLED}" --launch-type FARGATE --region us-east-1
+coveralls: test
+	pipenv run coverage lcov -o ./coverage/lcov.info
 
-run-prod: ##Runs the task in prod - see readme for more info
```

Comment:
lets put the real run-prod command in later, since prod hasn't been deployed yet.   Maybe just put it back in with a note that its a placeholder until its deployed (echo "Not yet in production")?

Reply:
Will do. Here's the ticket: https://mitlibraries.atlassian.net/browse/IN-890